lib/jekyll/configuration.rb

```diff
@@ -175,7 +175,7 @@ def read_config_file(file)
       next_config
     rescue SystemCallError
       if @default_config_file ||= nil
-        Jekyll.logger.warn "Configuration file:", "none"
+        Jekyll.logger.warn "Configuration file:", "none (looking for '#{file}')"
```

Comment:
@markstos I would disagree with that because you don't need a config file to run Jekyll at all. This check says "if you're looking for the default configuration file and haven't been told by the user it should it exist and it's not found, then just say there's no configuration". It's explicitly a non-error condition.